solang-server/src/serverutils/backend.rs

```diff
@@ -12,6 +12,16 @@ use solang::sema::*;
 
 use std::path::PathBuf;
 
+use solang::*;
+
+use solang::sema::ast::*;
+
+use solang::parser::pt;
+
+use solang::sema::ast::Expression::*;
+
+use solang::sema::ast::Type;
```

Comment:
use solang::sema::ast::*; already pulls in 
use solang::sema::ast::Type;
this line is not necessary

Reply:
done